pkg/generator/certificate.go

```diff
@@ -166,10 +166,49 @@ func GenerateSignedCert(rootCA *Certificate, algorithm v1alpha1.Algorithm, commo
 	if err != nil {
 		return cert, errors.WithStack(err)
 	}
-
 	return cert, nil
 }
 
+// GenerateSharedCertPEM generates a "non-conforming" certificate that's intended to not be used in PKI or any application unless explicitly designed to share keys.
+func GenerateSharedCertPEM(commonName string) ([]byte, []byte, []byte, error) {
```

Comment:
If the only difference is the date and the algo (RSA vs ECDSA) should be pass those as arguments to the function? We can set a default as ECDSA and 10years and you can pass "RSA" and time(0) in this special case.

Reply:
it's also not signed by the CA.

Reply:
got it